Frontend/src/views/Dashboard.vue

```diff
@@ -1,61 +1,145 @@
+    </v-card>
+
+    <v-alert :value="error" type="error" transition="scale-transition">{{error}}</v-alert>
   </div>
 </template>
 
 <script>
-import { signLaunch, submitLaunch } from '@/services/request';
+import Loading from "@/components/Dialogs/Loading.vue";
+import AppInfo from "@/components/Dialogs/AppInfo.vue";
+import { signLaunch, submitLaunch } from "@/services/request";
+import { apiURL } from "@/const.js";
+import axios from "axios";
 
 export default {
-  data () {
+  components: { Loading, AppInfo },
+  data() {
     return {
+      applications: [],
+      showInfo: false,
+      launchLoading: false,
       error: ""
+    };
+  },
+  watch: {
+    launchLoading(val) {
+      if (!val) return;
+      setTimeout(() => (this.launchLoading = false), 3000);
     }
   },
   methods: {
+    exitAppInfo(hideInfo) {
+      this.showInfo = hideInfo;
+    },
     launch(appId) {
       this.error = "";
 
-      signLaunch(appId).then(launchData => {
-        submitLaunch(launchData).then(launchResponse => {
-          window.location.href = launchResponse.redirectURL;
-        }).catch(err => {
+      signLaunch(appId)
+        .then(launchData => {
+          submitLaunch(launchData)
+            .then(launchResponse => {
+              window.location.href = launchResponse.redirectURL;
+            })
+            .catch(err => {
+              let code = err.response.status;
+
+              switch (code) {
+                case 500:
+                  this.error =
+                    "An unexpected server error occurred. Please try again momentarily.";
+                  break;
+                default:
+                  this.error =
+                    "An unexpected server error occurred. Please try again momentarily.";
+                  break;
+              }
+            });
+        })
+        .catch(err => {
           let code = err.response.status;
 
-          switch(code) {
+          switch (code) {
             case 500:
-              this.error = "An unexpected server error occurred. Please try again momentarily.";
+              this.error =
+                "An unexpected server error occurred. Please try again momentarily.";
               break;
             default:
-              this.error = "An unexpected server error occurred. Please try again momentarily.";
+              this.error =
+                "An unexpected server error occurred. Please try again momentarily.";
               break;
           }
-        })
-      }).catch(err => {
-        let code = err.response.status;
-
-        switch(code) {
-          case 500:
-            this.error = "An unexpected server error occurred. Please try again momentarily.";
-            break;
-          default:
-            this.error = "An unexpected server error occurred. Please try again momentarily.";
-            break;
-        }
-      })
+        });
+    }
+  },
+  async mounted() {
+    await axios
+      .get(`${apiURL}/api/applications`)
```

Comment:
this endpoint is incorrect, .../api/api/applications

Reply:
Sorry I didn't see this right away. It should be fixed now